.github/workflows/upstream_merge.yml

```diff
@@ -0,0 +1,38 @@
+name: Merge
```

Comment:
As discussed, please leave a comment here to tell downstream that this workflow can't be used with a strict merge policy (like Neb main uses). Something like: `This workflow cannot run if your fork uses review-based or CI-based branch protection (such as requiring a review before a merge), as the Github bot will not be able to bypass the protections to push directly to master.`

Reply:
![Screenshot_20210209-161026_GitHub.jpg](https://user-images.githubusercontent.com/46600554/107341265-56bfb900-6af1-11eb-8bd4-634b36bdbb21.jpg)